docs/3.tutorials/nfts/9-series.md

```diff
@@ -0,0 +1,428 @@
+    royalty: Option<HashMap<AccountId, u32>>,
+    // Set of tokens in the collection
+    tokens: UnorderedSet<TokenId>,
+    // What is the price of each token in this series? If this is specified, when minting,
+    // Users will need to attach enough $NEAR to cover the price.
+    price: Option<Balance>,
+    // Owner of the collection
+    owner_id: AccountId,
+}
+```
+
+This object stores information that each token will inherit from. This includes:
+- The metadata.
+- The royalties.
+- The price.
+
+If a price is specified, there will be no restriction on who can mint tokens in the series. In addition, if the `copies` field is specified in the metadata, **only** that number of NFTs can be minted. If the field is omitted, an unlimited amount of tokens can be minted.
+
+We've also added a field `tokens` which keeps track of all the token IDs that have been minted for this series. This allows us to deal with the potential `copies` cap by checking the length of the set. It also allows us to paginate through all the tokens in the series.
+
+### Series File
+
+Moving on to a new file that has been introduced, you'll notice that there's no longer the `mint.rs` file and it's been replaced by a newer `series.rs` file. This is responsible for everything related to creating series as well as minting NFTs. Starting with the logic for creating a new series, you'll see a new function `create_series`.
+
+```rust reference
+https://github.com/near-examples/nft-tutorial/blob/main/nft-series/src/series.rs#L7-L58
+```
+
+The function takes in a series ID in the form of a `u64`, the metadata, royalties, and the price for tokens in the series. It will then create the `Series` object and insert it into the contract's `series_by_id` data structure. It's important to note that the caller must be an approved creator and they must attach enough $NEAR to cover storage costs.
+
+#### Minting
+
+Next, we'll look at is the minting function. If you remember from before, this used to take a slew of parameters:
+- Token ID
+- Metadata
+- Receiver ID
+- Perpetual Royalties 
+
+With the new minting function, these parameters have been brought down to just two.
+- The series ID
+- The receiver ID.
+
+Notice how the token ID isn't required? This is because the token ID is automatically generated when minting. The ID is stored on the contract as `${series_id}:${token_id}` where the token ID is a nonce that increases each time a new token is minted in a series. This not only reduces the amount of information stored on the contract but it also acts as a way to check the specific edition number.
+
+The mint function will charge users for storage and if the series has a price, it will ensure that users attach enough $NEAR to cover the price **and** storage. At the end, it will then send all the $NEAR attached minus the storage costs to the **series owner**. It's important to note that in this case, the caller does **not** need to be an approved minter.
```

Comment:
This section is a little confusing. Let's take a stab at re-wording this to be a little more clear.